src/tools/kata-ctl/src/check.rs

```diff
@@ -129,6 +133,55 @@ pub fn run_network_checks() -> Result<()> {
     Ok(())
 }
 
+// Set of basic checks for kvm. Architectures should implement more specific checks if needed
+#[cfg(any(target_arch = "x86_64"))]
+pub fn check_kvm_is_usable_generic(kvm_device: &str, machine_type: i32) -> Result<()> {
```

Comment:
If this is architecture specific to x86, would it make sense to move most of this code into the arch specific module and have this function call a wrapper function in there.
cc @jodh-intel

Reply:
@amshinde it isn't, it isn't implemented for the architectures yet so I added this to avoid having the CI fail